CHANGELOG.md

```diff
@@ -488,17 +472,11 @@ and accepts and comma-separated list of namespaces as a string.
 
 ## 0.20.0 / 2018-06-05
 
-With this release we introduce a new Custom Resource Definition - the
-`PrometheusRule` CRD. It addresses the need for rule syntax validation and rule
-selection across namespaces. `PrometheusRule` replaces the configuration of
-Prometheus rules via K8s ConfigMaps. There are two migration paths:
+With this release we introduce a new Custom Resource Definition - the `PrometheusRule` CRD. It addresses the need for rule syntax validation and rule selection across namespaces. `PrometheusRule` replaces the configuration of Prometheus rules via K8s ConfigMaps. There are two migration paths:
```

Comment:
Is there no line length limit in mdox? I don't see why we would like to switch from nicely formatted multiline text to single-line one. Especially since the latter one is harder to compare when using `diff`.

Reply:
Hm, is it really hareder to review? 

Let us know if it's a blocker, we could add that feature, but it looks fine for me TBH.

![image](https://user-images.githubusercontent.com/6950331/127998684-1f9890da-a64b-4388-8de3-e9e9e7e1af45.png)

Reply:
Related https://github.com/Kunde21/markdownfmt/issues/29

Reply:
For me, yes, it is. With multi-line, GitHub would present just a small part that has changed, here it presents one giant block of changed text. I don't see why I would need to go through a 400 char paragraph to figure out the PR is just fixing a simple typo. Plus, if a paragraph is very long, GitHub will collapse file change in diff view.

Reply:
Hi @paulfantom! mdox now supports preserving soft line breaks([bwplotka/mdox#75](https://github.com/bwplotka/mdox/pull/75)). This PR has been updated with the same! 🙂